docs/user/document/references-usage-csaf_informational_advisory.en.md

```diff
@@ -1,3 +1,3 @@
 # References - Usage
 
-* For the `csaf_informational_advisory` profile at least one reference must exist which has links to an `external` source.
+For the `csaf_informational_advisory` profile at least one reference must exist which has links to an `external` source.
```

Comment:
Please add some details from the spec about the intended content of the reference.

Reply:
I added some explanation along the lines of the specification. Please check if that is suitable.